django_netjsonconfig/api/generics.py

```diff
@@ -10,16 +29,16 @@ class BaseSearchTemplate(ListAPIView):
     def get_queryset(self):
         name = self.request.GET.get('name', None)
         des = self.request.GET.get('des', None)
-        temp = self.template_model.filter(sharing='public')
+        template = self.template_model.objects.filter(sharing='public')
```

Comment:
mm sorry I didn't read the code properly, this variable contains a `queryset`, so it can be named `queryset` or simply `qs`